src/core/src/consensus_state.rs

```diff
@@ -52,14 +51,14 @@ pub struct PopulationSample {
 *     block
 *
 */
-#[derive(Clone, Debug, PartialEq, Default)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Default)]
 pub struct EstimateInfo {
     pub population_estimate : f64,
-    pub previous_block_id: BlockId,
+    pub previous_block_id: String,//BlockId,
```

Comment:
Add context for  BlockId comment

Reply:
Updated context for both the fields BlockId and PeerId